CONTRIBUTING

```diff
@@ -99,3 +99,63 @@ Pull Request Review
 Please note that there is a minimum review period of 24 hours for any patch. The purpose of this rule is to ensure that all interested parties have an opportunity to review every patch. When posting a patch before or after a holiday break it is important to extend this period as appropriate.
 
 All subsequent changes made to patches must be force-pushed to the PR branch before merging it into the main branch.
+
+Examples
```

Comment:
Maybe to change ``Examples`` to ``Merging guidelines``?

The Contribution file is now so diverse that I don't know to which part are these examples belongs.